docs/pages/enroll-resources/kubernetes-access/kubernetes-access-examples.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: Examples of Role-Based Access Control in Kubernetes
+description: RBAC examples with Kubernetes.
+---
+
+In Teleport's flexible RBAC system, you can define custom roles and granularly control what users can do within your Kubernetes clusters. 
+Limit access to specific namespaces, enforce read-only permissions, or restrict administrative actions as needed.
+
+:Note:
+Please reference [Setting Up Teleport Access Controls for Kubernetes](../../enroll-resources/kubernetes-access/manage-access.mdx) before proceeding, if you have not done so already.
+
+Here are a few common use cases. These are sample examples to get you started; for more in-depth coverage of roles in Kubernetes access, please visit the 
+[Access Controls guide](./controls.mdx) for Kubernetes.
+
+## Use case 1: Example role for developers
+
+In many organizations, developers need limited access to Kubernetes clusters, typically restricted to their development environments. 
+
+Below is an example of a role that provides developers with access to only the dev namespace:
+
+```code
```

Comment:
```suggestion
```yaml
```

Reply:
We should use `yaml` for other YAML examples as well so syntax highlighting works.